src/app/shared/ag-grid-editors/numeric-cell-editor/numeric-cell-editor.component.html

```diff
@@ -1,3 +1 @@
-<div>
-    <input matInput [min]="minValue" [max]="maxValue" #input type="number" [(ngModel)]="value">
-</div>
+<input matInput type="number" [min]="minValue" [max]="maxValue" #input [(ngModel)]="value">
```

Comment:
Please check what happens if we use `mat-form-field`?
Also, check if we can fix the input arrows issue for Firefox.

Reply:
The firefox issue is a known bug: https://bugzilla.mozilla.org/show_bug.cgi?id=344616